samples/audio/sof/CMakeLists.txt

```diff
@@ -2,6 +2,13 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
+set(sof_module $ENV{ZEPHYR_BASE}/../modules/audio/sof)
+
+if (NOT ${BOARD} MATCHES "^intel_adsp_cavs.*")
```

Comment:
Can we have a comment here stating that this enables an arch generic build and will be updated as xtos is deprecated and SOF drivers are moved into zephyr.